python/doc/theory/numerical_methods/nataf_transformation.rst

```diff
@@ -3,40 +3,35 @@
 Generalized Nataf Transformation
```

Comment:
I really do not know... I verified: no problem in the numerical_methods.rst file.....

Reply:
I try to see why.

Reply:
Ok. Got it. I used destructive bisection to find the issue: i removed half of the content of the file until the issue was gone. The issue is that the file `python/doc/theory/numerical_methods/nataf_transformation.rst` contains the next string:
```
Generalized Nataf Transformation
--------------------------------
```
then:
```
Generalized Nataf transformation:
---------------------------------
```
But the `---` defines the title of a section in RST (see [quickstart](https://docutils.sourceforge.io/docs/user/rst/quickstart.html)). Hence both sections are mentioned in the main page. The solution is to replace the second one with:
```
Generalized Nataf transformation
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
because `===` defines a sub-section. But this creates a document with one single sub-section, which is not consistent on my opinion. So I suggest to create an "Introduction" and then the second sub-section. This produces:
![image](https://user-images.githubusercontent.com/31351465/235185441-b50d6b96-e7ec-4a48-90c6-79b4c38e3cec.png)

I make detailed suggestion to get to this.

Reply:
Thank you for getting it.... I committed all your suggestions!